src/main/java/com/google/finapp/SpannerDao.java

```diff
@@ -31,7 +31,8 @@
     this.databaseClient = databaseClient;
   }
 
-  void createCustomer(ByteArray customerId, String name, String address) throws SpannerException {
+  @Override
+  public void createCustomer(ByteArray customerId, String name, String address) throws SpannerException {
```

Comment:
If the server code which uses this were in a separate package, we'd need these methods to be public. We'd want to do that in future I suppose for better code organization, but with the current state of packages, we don't need this to be public.

Reply:
i think this happened as a result of creating the interface (an intellij feature), so i'll resolve this

Reply:
if i dont specify that the methods are public, i get build failures. 
![image](https://user-images.githubusercontent.com/51244558/125534173-f4c047ff-a796-4870-80b9-2ace0b323fac.png)